### PR TITLE
chore: correct repository for marketplace

### LINKED ISSUE #1: repo url in vscode
Hi,

The url in visual studio code points to `https://github.com/git@github.com:YusukeHirao/vscode-markuplint`, which gives a 404. It should probalby be `https://github.com/yusukehirao/vscode-markuplint`.

Hope this helps, Marc

### PR DESCRIPTION
**Pull request by JounQin**
close #1

### CODE REVIEW

**Comment by JounQin** [comment]
friendly ping @YusukeHirao

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	"version": "1.8.0",
 	"publisher": "yusukehirao",
 	"license": "MIT",
-	"repository": "git@github.com:YusukeHirao/vscode-markuplint.git",
+	"repository": "https://github.com/YusukeHirao/vscode-markuplint.git",
```
Comment (YusukeHirao) [inline package.json:8]: The user in the path is incorrect.
```diff
- https://github.com/YusukeHirao/vscode-markuplint.git
+ https://github.com/markuplint/vscode-markuplint
```

Comment (JounQin) [reply package.json]: Done!

Comment (YusukeHirao) [reply package.json]: Oh, sorry about `.git`.

**Pull request merged**